test/e2e/bundle_e2e_test.go

```diff
@@ -70,9 +73,19 @@ var _ = Describe("Installing bundles with new object types", func() {
 
 			// Wait for the Subscription to succeed
 			Eventually(func() error {
-				_, err = fetchSubscription(operatorClient, testNamespace, subName, subscriptionStateAtLatestChecker)
+				sub, err := fetchSubscription(operatorClient, testNamespace, subName, subscriptionStateAtLatestChecker)
```

Comment:
hmm, fetchSubscription already polls, so I think the `Eventually` matcher is redundant here.